cluster/manifests/ci/base/templates/build-container-template.yaml

```diff
@@ -34,6 +34,34 @@ spec:
 
         SERVICES=$(builder --changed --status=pending --sha={{inputs.parameters.sha}})
         echo $SERVICES
+  - name: checkout-tf
```

Comment:
I don't think you need to do a git checkout since you have the volume mount that the first git checkout used to clone the repo

Reply:
@jperson yeah that's true. I just changed it to grab the terraform changes, not checkout or fetch.